resources/views/teachers/show.blade.php

```diff
@@ -0,0 +1,64 @@
+@extends('layouts.app')
+
```

Comment:
Maybe before the section content you can add the Facebook Og tags..
check out view/posts/show.blade.php
The section called..
@section('fb-tags')